apple/swift-docc-render
# Prevent Safari from reading the navigator group markers as clickable

Bug/issue #, if applicable: 

## Summary

VO on Safari reads any element with an event listener attached as a clickable element.

To prevent that we have to remove the listener, or add a `role`. - https://pauljadam.com/demos/voiceover-clickable.html

We cannot assign `role` on the Reference, because that is an actual prop it has.

That is why I created a simple `OptionalLink` component, that renders a span instead of a `router-link` if `disabled` or no url is passed. This is a very very basic version of the

## Dependencies

NA

## Testing

Steps:
1. Make sure links in the navigator are passing over query params properly
2. Make sure labels are not read out as `clickable` in Safari by VO

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
